cmd/sriovdp/sriov-device-plugin.go

```diff
@@ -493,20 +511,24 @@ func (sm *sriovManager) GetDevicePluginOptions(ctx context.Context, empty *plugi
 func (sm *sriovManager) Allocate(ctx context.Context, rqt *pluginapi.AllocateRequest) (*pluginapi.AllocateResponse, error) {
 	resp := new(pluginapi.AllocateResponse)
 	pciAddrs := ""
+	id := ""
```

Comment:
Why the var `id` moved up instead of local to the loop? I don't see it's being used outside of the loop anyway. Maybe I'm missing something?

Reply:
fixed